diff -urN gnome-mag-0.10.8.orig/configure.in gnome-mag-0.10.8/configure.in
--- gnome-mag-0.10.8.orig/configure.in	2004-03-08 21:08:01.977528704 +0100
+++ gnome-mag-0.10.8/configure.in	2004-03-08 21:08:21.816512720 +0100
@@ -43,7 +43,7 @@
 AM_DISABLE_STATIC
 AM_PROG_LIBTOOL
 
-ALL_LINGUAS="ar az be bg bn br ca cs cy da de el en_CA es et eu fa fi fr ga he hi hr hu is it ja ko lt lv mn ms nl nn no pa pl pt pt_BR sk sl sq sr sr@Latn sv th tk tr uk vi zh_CN zh_TW"
+ALL_LINGUAS="ar az be bg bn br ca cs cy da de el en_CA es et eu fa fi fr ga he hi hr hu is it ja ko lt lv mn ms nb nl nn pa pl pt pt_BR sk sl sq sr sr@Latn sv th tk tr uk vi zh_CN zh_TW"
 GETTEXT_PACKAGE=gnome-mag
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [The gettext package])
