diff -urN gnome-mag-0.15.7/colorblind/keybinder/Makefile.am gnome-mag-0.15.7.new/colorblind/keybinder/Makefile.am
--- gnome-mag-0.15.7/colorblind/keybinder/Makefile.am	2009-06-29 19:05:13.000000000 +0200
+++ gnome-mag-0.15.7.new/colorblind/keybinder/Makefile.am	2009-07-02 22:16:31.858917473 +0200
@@ -1,6 +1,6 @@
 INCLUDES = $(PYTHON_INCLUDES) $(CBAPPLET_CFLAGS)
 
-_keybinderdir = $(pythondir)/colorblind/keybinder
+_keybinderdir = $(pyexecdir)/colorblind/keybinder
 _keybinder_LTLIBRARIES = _keybinder.la
 _keybinder_la_LDFLAGS = -module -avoid-version -export-symbols-regex init_keybinder
 _keybinder_la_LIBADD = $(CBAPPLET_LIBS)
diff -urN gnome-mag-0.15.7/colorblind/Makefile.am gnome-mag-0.15.7.new/colorblind/Makefile.am
--- gnome-mag-0.15.7/colorblind/Makefile.am	2009-06-29 19:05:13.000000000 +0200
+++ gnome-mag-0.15.7.new/colorblind/Makefile.am	2009-07-02 22:17:06.768913923 +0200
@@ -1,12 +1,12 @@
 SUBDIRS = ui osutils data keybinder
 
 colorblind-applet: colorblind-applet.py
-	sed -e "s|\@PYTHONDIR\@|$(pythondir)|" $< > $@
+	sed -e "s|\@PYTHONDIR\@|$(pyexecdir)|" $< > $@
 
 colorblindbindir = $(libdir)/colorblind-applet
 colorblindbin_SCRIPTS = colorblind-applet
 
-colorblinddir = $(pythondir)/colorblind
+colorblinddir = $(pyexecdir)/colorblind
 
 colorblind_PYTHON = \
 	__init__.py \
diff -urN gnome-mag-0.15.7/colorblind/osutils/Makefile.am gnome-mag-0.15.7.new/colorblind/osutils/Makefile.am
--- gnome-mag-0.15.7/colorblind/osutils/Makefile.am	2009-06-29 19:05:13.000000000 +0200
+++ gnome-mag-0.15.7.new/colorblind/osutils/Makefile.am	2009-07-02 22:17:28.432234329 +0200
@@ -1,6 +1,6 @@
 INCLUDES = $(PYTHON_INCLUDES) $(CBAPPLET_CFLAGS)
 
-_osutilsdir = $(pythondir)/colorblind/osutils
+_osutilsdir = $(pyexecdir)/colorblind/osutils
 _osutils_LTLIBRARIES = _osutils.la
 _osutils_la_LDFLAGS = -module -avoid-version init_osutils
 _osutils_la_LIBADD = $(CBAPPLET_LIBS)
diff -urN gnome-mag-0.15.7/colorblind/ui/Makefile.am gnome-mag-0.15.7.new/colorblind/ui/Makefile.am
--- gnome-mag-0.15.7/colorblind/ui/Makefile.am	2009-06-29 19:05:13.000000000 +0200
+++ gnome-mag-0.15.7.new/colorblind/ui/Makefile.am	2009-07-02 22:17:37.372340129 +0200
@@ -1,4 +1,4 @@
-colorblinddir = $(pythondir)/colorblind/ui
+colorblinddir = $(pyexecdir)/colorblind/ui
 colorblind_PYTHON = \
 	About.py \
 	ColorblindPreferencesUI.py \
